addons/stock/models/stock_move_line.py

```diff
@@ -520,6 +520,13 @@ def _get_similar_move_lines(self):
             lines |= picking_id.move_line_ids.filtered(lambda ml: ml.product_id == self.product_id and (ml.lot_id or ml.lot_name))
         return lines
 
+    def _get_value_production_lot(self):
+        return {
+                    'company_id': self.company_id.id,
```

Comment:
```suggestion
            'company_id': self.company_id.id,
```
(same for next fields)